### PR TITLE
[Dev] Revert misattributed license headers

### PR DESCRIPTION
**Pull request by keveleigh**
Overview
---
Reverts misattributed license headers. Also, removes some copyright headers that were added incorrectly.

Changes
---
- Fixes: #1931 in dev.

### CODE REVIEW

File: Assets/MixedRealityToolkit-Examples/Sharing/Scripts/UserNotifications.cs [hunk 1]
```diff
@@ -1,7 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
-
-using MixedRealityToolkit.Sharing;
+﻿using MixedRealityToolkit.Sharing;
```
Comment (davidkline-ms) [inline Assets/MixedRealityToolkit-Examples/Sharing/Scripts/UserNotifications.cs]: Add the "Licensed under.." comment

File: Assets/MixedRealityToolkit-Examples/UX/Scripts/ExampleData/NoteDataProvider.cs [hunk 2]
```diff
@@ -1,7 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
-
-using MixedRealityToolkit.Examples.UX.Controls;
+﻿using MixedRealityToolkit.Examples.UX.Controls;
```
Comment (davidkline-ms) [inline Assets/MixedRealityToolkit-Examples/UX/Scripts/ExampleData/NoteDataProvider.cs]: Add the "Licensed under.." comment

File: Assets/MixedRealityToolkit-Examples/UX/Scripts/ExampleData/NoteDataProvider.cs [hunk 3]
```diff
@@ -1,7 +1,4 @@
...
 using System.Collections.Generic;
```
Comment (davidkline-ms) [inline Assets/MixedRealityToolkit-Examples/UX/Scripts/ExampleData/NoteDataProvider.cs]: Add the "Licensed under.." comment

File: Assets/MixedRealityToolkit-Examples/UX/Scripts/InteractiveGroup.cs [hunk 4]
```diff
@@ -1,7 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
-
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
```
Comment (davidkline-ms) [inline Assets/MixedRealityToolkit-Examples/UX/Scripts/InteractiveGroup.cs]: Add the "Licensed under.." comment

File: Assets/MixedRealityToolkit/Common/Scripts/Extensions/EnumerableExtensions.cs [hunk 5]
```diff
@@ -1,7 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
-
-using System;
+﻿using System;
```
Comment (davidkline-ms) [inline Assets/MixedRealityToolkit/Common/Scripts/Extensions/EnumerableExtensions.cs]: Add the "Licensed under.." comment

File: Assets/MixedRealityToolkit/SpatialMapping/Shaders/SpatialMappingTap.shader [hunk 6]
```diff
@@ -1,6 +1,3 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
-
 Shader "Spatial Mapping/Spatial Mapping Tap"
```
Comment (davidkline-ms) [inline Assets/MixedRealityToolkit/SpatialMapping/Shaders/SpatialMappingTap.shader]: Add the "Licensed under.." comment

**Comment by StephenHodgson** [comment]
A good amount of these classes were written by me, and can be reverted to the normal license header. I think the only two classes that needed to be addressed was `XdeGuestLocator.cs` and `BuildAndDeployWindow.cs`

**Pull request merged**